test/functional/cmdLineTests/criu/criuCatVlog.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+#
+# Copyright IBM Corp. and others 2023
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] https://openjdk.org/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+#
+
+echo "start running script";
+# the expected arguments are:
+# $1 is the VLOG_TO_CAT
+# $2 is the CHECK_PREVIOUS_TEST_OUTPUT
+# $3 is the DELETE_PREV_TEST_OUTPUT
+
+if [ -f "$1" ]; then
+    echo "Outputting vlog $1"
+    cat $1
+    echo ""
+
+    if [ "$2" == true ]; then
+        echo "Outputting previous test output"
+        cat testOutput
+        echo ""
+    fi
+else
+    if [ "$2" == true ]; then
+        echo "Checking previous test output"
+        cat testOutput
+        echo ""
+
```

Comment:
This code looks like duplicated, can we change the if logic to combine them together?

Reply:
Done in [force push](https://github.com/eclipse-openj9/openj9/compare/61d947d1860b951a1ce8a130fd331f547014fe5e..c4640eb70c622558460a107ead38a3290baad0f6).